File: src/main/java/com/blakgeek/akka/java/pingpong/PingPong.java
package com.blakgeek.akka.java.pingpong;

import akka.actor.ActorSystem;
import akka.actor.Props;

/**
 * User: Carlos Lawton
 * Date: 3/20/14
 * Time: 1:45 PM
 */
public class PingPong {

    public static void main(String[] args) {

        ActorSystem stadium = ActorSystem.create();
        stadium.actorOf(Props.create(Game.class, 5));
    }
}

File: scala/src/main/scala/com/blakgeek/akka/scala/pingpong/PingPong.scala
package com.blakgeek.akka.scala.pingpong

import akka.actor.{Props, ActorSystem}

/**
 * User: Carlos Lawton
 * Date: 3/21/14
 * Time: 2:24 PM
 */
object PingPong extends App {

  val system = ActorSystem.create()
  // create the to players
  val ping = system.actorOf(Props(classOf[Player], "ping"), name = "ping")
  val pong = system.actorOf(Props(classOf[Player], "pong"), name = "pong")
  // tell on of them its there serve and who they are server to
  ping ! YourServe(pong, Score(0,0))
}
